virtual_ship/expedition/simulate_measurements.py

```diff
@@ -0,0 +1,104 @@
+"""simulate_measurements function."""
+
+from datetime import timedelta
+from pathlib import Path
+
+from ..instruments.adcp import simulate_adcp
+from ..instruments.argo_float import simulate_argo_floats
+from ..instruments.ctd import simulate_ctd
+from ..instruments.drifter import simulate_drifters
+from ..instruments.ship_underwater_st import simulate_ship_underwater_st
+from .input_data import InputData
+from .ship_config import ShipConfig
+from .simulate_schedule import MeasurementsToSimulate
+
+
+def simulate_measurements(
+    expedition_dir: str | Path,
+    ship_config: ShipConfig,
+    input_data: InputData,
+    measurements: MeasurementsToSimulate,
+) -> None:
+    """
+    Simulate measurements using parcels.
```

Comment:
```suggestion
    Simulate measurements using Parcels.
```
(we started using capitalised Parcels again...)